packages/core-mobile/app/new/routes/signup.tsx

```diff
@@ -0,0 +1,199 @@
+import { View, Button, useTheme, SafeAreaView, Logos } from '@avalabs/k2-alpine'
+import React, { useEffect } from 'react'
+import { useSelector } from 'react-redux'
+import {
+  selectIsSeedlessOnboardingAppleBlocked,
+  selectIsSeedlessOnboardingBlocked,
+  selectIsSeedlessOnboardingGoogleBlocked
+} from 'store/posthog'
+import { useSeedlessRegister } from 'seedless/hooks/useSeedlessRegister'
+import SeedlessService from 'seedless/services/SeedlessService'
+import { MFA } from 'seedless/types'
+import AppleSignInService from 'services/socialSignIn/apple/AppleSignInService'
+import GoogleSigninService from 'services/socialSignIn/google/GoogleSigninService'
+import { OidcProviders } from 'seedless/consts'
+import { hideLogoModal, showLogoModal } from 'new/components/LogoModal'
+import { router } from 'expo-router'
+import AnalyticsService from 'services/analytics/AnalyticsService'
+import { showSnackbar } from 'new/utils/toast'
+
+export default function Index(): JSX.Element {
+  const { theme } = useTheme()
+  const isSeedlessOnboardingBlocked = useSelector(
+    selectIsSeedlessOnboardingBlocked
+  )
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { register, isRegistering, verify } = useSeedlessRegister()
+
+  useEffect(() => {
+    isRegistering ? showLogoModal() : hideLogoModal()
+  }, [isRegistering])
+
+  const handleSignupWithMnemonic = (): void => {
+    // todo: CP-9604
+    // navigate(AppNavigation.Onboard.Welcome, {
+    //   screen: AppNavigation.Onboard.AnalyticsConsent,
+    //   params: {
+    //     nextScreen: AppNavigation.Onboard.CreateWalletStack
+    //   }
+    // })
+    // AnalyticsService.capture('RecoveryPhraseClicked')
+  }
+
+  const handleAccessExistingWallet = (): void => {
+    router.navigate('./accessWallet/')
```

Comment:
just curious, if there difference between `./accessWallet/` vs `./accessWallet`

Reply:
There is no difference I think, but I removed the trailing /.